### PR TITLE
Add test scripts

### PR DESCRIPTION
**Pull request by elv-peter**
live stream, media content master, mez, abr playout master submit

### CODE REVIEW

File: testScripts/PlayoutObject.js [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+    const pathlist = await ReadDir(dir);
+    var mediaStructFile;
+
+    try {
+
+      for (var i = 0, len = pathlist.length; i < len; i++) {
+	var singleFilePath = pathlist[i];
+
+	console.log("Reading file: " + singleFilePath + " ...");
+	var fileContents = fs.readFileSync(singleFilePath);
```
Comment (lukseven) [inline testScripts/PlayoutObject.js]: How large are these files? We shouldn't read them into memory, but upload them using a stream/reader.

File: testScripts/PlayoutObject.js [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+        // upload file as part
+        var args = {
+          libraryId: this.libId,
+	  objectId: this.objectId,
+          writeToken: this.writeToken,
+          data: fileContents,
+	  encryption: contentEncryption
+        };
+        console.log("Uploading part for " + filename);
+        var uploadPartResult = await this.elvclient.UploadPart(args);
```
Comment (lukseven) [inline testScripts/PlayoutObject.js]: Are our media objects still using parts? We should move to using files in bundles...

**Comment by elv-peter** [comment]
@elv-serban can comment on PlayoutObject I think

**Comment by elv-peter** [comment]
I'm going to make a new branch and PR with just SetupLive.js. The other files are probably not useful anymore

**Pull request closed without merging**